django/db/migrations/state.py

```diff
@@ -443,6 +443,11 @@ def from_model(cls, model, exclude_rels=False):
                 elif name == "index_together":
                     it = model._meta.original_attrs["index_together"]
                     options[name] = set(normalize_together(it))
+                elif name == "indexes":
+                    indexes = model._meta.indexes
```

Comment:
You need to copy the `model._meta.indexes` list, otherwise you're reusing the same index instance across multiple models which could lead to other errors. I suggest `indexes = model._meta.indexes[:]`.